src/game_party.cpp

```diff
@@ -108,12 +108,12 @@ int Game_Party::GetItemCount(int item_id, bool get_equipped) {
 
 void Game_Party::GainGold(int n) {
 	data.gold = data.gold + n;
-	data.gold = std::min(std::max(data.gold, 0), 999999);
+	data.gold = std::min(std::max(data.gold, (int32_t)0), (int32_t)999999);
```

Comment:
These casts look ugly. `min/max` can use the common type of the arguments, but needs explicit type then, i.e.
`max<int32_t>(some, value);` Maybe is better.

Reply:
Thanks, changed this.